content/post/2017-08-28-adding-citations-to-posts.html

```diff
@@ -48,16 +39,24 @@
 </div>
 <div id="example-2-adding-cites-for-r-packages-using-knitr" class="section level1">
 <h1>Example 2: Adding cites for R packages using <code>knitr</code></h1>
-<p>There is a helper function <code>write_bib()</code> in <code>knitr</code> to generate BibTeX entries automatically for R packages. You can use this method to add citations in RMarkdown posts in <strong>blogdown</strong>. The difference is that instead of saving your <code>.bib</code> file(s) in the <code>content\post\</code> directory, you will use the <code>write_bib</code> function from the <code>knitr</code> package within an R chunk. This will create a <code>.bib</code> file for you in the <code>content\post\</code> directory.</p>
+<p>There is a helper function <code>write_bib()</code> in <code>knitr</code> to generate BibTeX entries automatically for R packages. You can use this method to add citations in RMarkdown posts in <strong>blogdown</strong>. The difference is that instead of saving your <code>.bib</code> file(s) in the <code>content/post/</code> directory, you will use the <code>write_bib</code> function from the <code>knitr</code> package within an R chunk. This will create a <code>.bib</code> file for you in the <code>content/post/</code> directory.</p>
 <pre class="r"><code># list specific packages
 knitr::write_bib(c(&quot;package1&quot;, &quot;package2&quot;), &quot;packages.bib&quot;)</code></pre>
 <pre class="r"><code># or include all used in the post
 knitr::write_bib(.packages(), &quot;packages.bib&quot;) </code></pre>
```

Comment:
Could you add `warning=FALSE` to the R source code chunk to avoid the following warning messages?

Reply:
I'll do both of these changes tomorrow when I work on that post- is it OK if both are in a different PR?

Reply:
I don't mind if the two changes are in this PR or a separate PR.